### DISCUSSION TITLE
Help for people transitioning from homebridge-plugin-eufy-security

### DISCUSSION THREAD

**Discussion by PeterAlexander75** [samemory/homebridge-eufy-security-control#21] [Q&A]
I know a number of people are looking at moving away from https://github.com/birkir/homebridge-plugin-eufy-security so I thought it would be useful to have a thread to help out.

Feel free to ask any questions - I've made the move and happy to help out.

**Comment by jeffd59** [comment]
Hello 
I would like to use the plugin for my 2K eufy cam.
Thank you

**Reply by samemory** [reply]
Which model? I believe the outdoor 2K cams have native HomeKit support.

**Reply by jasonwfrancis1** [reply]
yeah cameras should I thought, just not doorbells.

**Reply by mmz06** [reply]
It seems to work well, by many aspects like Homebase, arming configuration, and Doorbell 2K video, but it doesn't provide static snapshots for it, and no notification when it's pressed.
But overall, it seems to be like a really good start.

**Reply by lenoxys** [reply]
We choose to not put the camera stream and snapshot since you can already do it through the official Eufy method.

If you need it can you feed a feature request so we can track it ?
